client/src/components/pages/PageNotFound.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+class Landing extends Component {
+    render() {
+        return (
+            <div className="landing-container">
+                <div className="landing-header">
+                    <h1 className="landing-title">404</h1>
+                    <h1>Page not found</h1>
+                    <p>If you believe this occured in error, please contance support.</p>
```

Comment:
*contact. Also put our email on this page?

Reply:
fixed and added